lib/sandbox/timers.js

```diff
@@ -44,17 +44,26 @@ var /**
      */
     defaultTimers = timerFunctionNames.reduce(function (timers, name) {
         // get hold of default timer functions as available in global scope
-        var fnset = (new Function('return (typeof set' + name + // eslint-disable-line no-new-func
-                ' === "function" ? set' + name + ' : undefined);'))(),
-            fnclr = (new Function('return (typeof clear' + name + // eslint-disable-line no-new-func
-                ' === "function" ? clear' + name + ' : undefined);'))();
+        var setFnName = 'set' + name,
+            clrFnName = 'clear' + name,
+            clrAllFnName = 'clearAll' + name + 's',
+            fnset = (new Function('return (typeof ' + setFnName + ' === "function" ? ' + // eslint-disable-line no-new-func
+                setFnName + ' : undefined);'))(),
+            fnclr = (new Function('return (typeof ' + clrFnName + ' === "function" ? ' + // eslint-disable-line no-new-func
+                clrFnName + ' : undefined);'))(),
+            fnclrAll = (new Function('return (typeof ' + clrAllFnName + ' === "function" ? ' + // eslint-disable-line no-new-func
```

Comment:
@shamasis could you review this part in specific? I've added `clearAll` timers to `defaultTimers`.